Assets/Scripts/TargetFollower.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TargetFollower : MonoBehaviour 
+{
+    [SerializeField]
+    private float followSpeed;
+
+    [SerializeField]
+    private Transform targetTransform;
+
+    void FixedUpdate()
+	{
+        transform.position = Vector3.Lerp(transform.position, targetTransform.position, followSpeed * Time.deltaTime);
```

Comment:
Nogmaals, ookal is het 1 line. "FixedUpdate", zegt vrijwel niks over de logica dat hier gebeurd. Het is altijd beter om een aparte functie te maken die uitleg geeft over de logica.